src/neuroglancer/async_computation/request.ts

```diff
@@ -34,7 +34,7 @@ function returnWorker(worker: Worker) {
 }
 
 function getNewWorker(): Worker {
-  const worker = new Worker('async_computation.bundle.js');
+  const worker = new Worker('/async_computation.bundle.js');
```

Comment:
Since this is loaded from another worker, I believe the path is resolved relative to that worker's path, rather than the main document.  Therefore, I think you don't need to change this.